SCSS/Imports/_general.scss

```diff
@@ -27,4 +27,8 @@ body {
 
 .button:focus{
   box-shadow: 5px 5px 2px $red, -5px -5px 2px $red, 5px -5px 2px $red, -5px 5px 2px $red;
+}
+
+h1{
+  font-size: 1.25em;
```

Comment:
This is global so the one of the h1 font-sizes might be overriding the other displayCase vs general :)

Reply:
Also, could capynotasaurus be slightly bigger? It looks like there is still room on mobile view to be a lil bigger and for those visually impaired such as myself it would be nice to be a lil larger. At the mo h1 looks very similar to the dinosaur name h2's :)